ern-container-gen-android/test/convertToAndroidX-test.ts

```diff
@@ -0,0 +1,19 @@
+import { expect } from 'chai';
+import { AndroidGenerator } from 'ern-container-gen-android';
+import { shell, createTmpDir } from 'ern-core';
+import path from 'path';
+
+describe('Test Convert files in the directory to AndroidX Imports Logic', () => {
+  const tmpDir = createTmpDir();
+  const fixturesPath = path.join(__dirname, 'fixtures');
+
+  shell.cp('-rf', fixturesPath, tmpDir);
+
+  it('Expect files in directory to be converted to AndroidX', () => {
```

Comment:
Can you slightly reword these test cases, so that they read like a sentence? For example:
`it('converts files in a directory to AndroidX', ..`
Similar below: `it('returns ...`.